### ISSUE TITLE
Ubuntu 23.10 - working as expected!

### ISSUE THREAD

**Issue by b0iii2c0de** [kumakichi/Deepl-linux-electron#15]
Hi there, mate!

First of all thank you for your time and efforts to deliver such good thing fo linux! =)

I've downloaded the last 1.3.1 version and it works just fine! Would you mind if I humbly ask you to add "Dark Theme", pls? Because the default skin burn outs one's eyes! Looking forward for new badass needed feature, thnx in advance!

**Comment by b0iii2c0de** [comment]
And I found that feature existing in mac/win native apps is lacking in you beautiful app, I mean a when you select a text then press shortcut that call small modal window with translation, not the whole app.
Is this possible to add this feature as well, in next iteration of your app for linux? 😊

**Comment by kumakichi** [comment]
> Hi there, mate!
> 
> First of all thank you for your time and efforts to deliver such good thing for linux! =)
> 
> I've downloaded the last 1.3.1 version and it works just fine! Wouldn't you mind if I humbly ask you to add "Dark Theme", pls? Because the default skin burn outs one's eyes! Looking forward for new badass needed feature, thnx in advance!

I don't think this should be too much trouble, I'll see if I have time to try it these days

> And I found that feature existing in mac/win native apps is lacking in you beautiful app, I mean a when you select a text then press shortcut that call small modal window with translation, not the whole app. Is this possible to add this feature as well, in next iteration of your app for linux? 😊

This is a bit tricky and won't be implemented currently

**Comment by b0iii2c0de** [comment]
Hi there! At least "Dark Theme" would make my day and save my eyes! Thank you! Looking forward for this new feature!

**Comment by b0iii2c0de** [comment]
Dark theme working as expected on Ubunt 23.10! Thnx, @kumakichi !
